decidim/decidim
# Fix Proposals bulk action form

#### :tophat: What? Why?
When sending a bulk action -form for Proposals in the Admin view, to update the category of a proposal for example, it doesn't clear the selection of the proposals for the form. This causes consecutive bulk action -updates for different proposals to affect the previously selected proposal even though that proposal's checkbox was not ticked.

This is a bug which can cause mistakes without noticing them when updating different proposals in the admin panel with bulk actions.

Also fixes a bug where the number of proposals selected doesn't reset after updating the proposals.

#### Testing
1. Go to the Decidim Admin panel, move to the proposal listing of some Proposals -component.
2. Tick a checkbox of proposal ***"x"*** and press the "actions" -button.
3. Choose any bulk action, for example "Change category".
4. Pick a category from the list and press "Update".
5. The checkboxes should be empty in all of the proposals on the list, pick a new ***different*** proposal ***"y"*** and tick its box.
6. Do the same category change for that proposal with a different category than what you picked for ***"x"***, update, and notice that the category you just picked for proposal ***"y"*** is also set for the first proposal ***"x"***.

### :camera: Screenshots
BUG :
![image](https://github.com/decidim/decidim/assets/110532525/0c2d3302-f084-4787-a986-319964c6a577)

![image](https://github.com/decidim/decidim/assets/110532525/6dc32113-00b0-4f0d-8279-d8b32dea0506)

![image](https://github.com/decidim/decidim/assets/110532525/3d401759-de75-4366-b2d2-bbf828a6f70d)



:hearts: Thank you!
